util/paywall.go

```diff
@@ -128,3 +129,13 @@ func PayWithTestnetFaucet(faucetURL string, address string, amount float64, over
 
 	return fr.Txid, nil
 }
+
+// ProcessPaywallEnforcement checks that a user has paid the paywall
+func VerifyUserPaid(user *database.User, paywallAmount float64) bool {
+	if user.NewUserPaywallAddress == "" &&
```

Comment:
I don't think we should check the `NewUserPaywallAmount` against the config option; it should only check that `NewUserPaywallAddress == ""`. If we ever want to ensure the user pays exactly the amount specified by the current config option, we would just ignore or remove the `NewUserPaywallAmount` field from the user database.

Reply:
fixed